Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml

```diff
@@ -225,3 +225,25 @@
       seed: poppy
     - type: Sprite
       sprite: Objects/Specific/Hydroponics/poppy.rsi
+
+- type: entity
+  parent: SeedBase
+  name: packet of lingzhi seeds
+  description: "Also known as reishi."
+  id: LingzhiSeeds
+  components:
+    - type: Seed
+      seed: lingzhi
+    - type: Sprite
+      sprite: Objects/Specific/Hydroponics/lingzhi.rsi
+
+- type: entity
+  parent: SeedBase
+  name: packet of ambrosia vulgaris seeds
+  description: "A medicinal plant for the common folk."
+  id: AmbrosiaVSeeds ##The megaseed servitor won't load these if they have a fuller name and I have no idea why
```

Comment:
..what

Reply:
did the longer name you were trying to use already exist?

Reply:
No